### ISSUE TITLE
[Feature Request] One Line Zip Code Super Challenge Blast, fitting a challenge into One Memo Code

### ISSUE THREAD

**Issue by ackza** [ChallengeDAC/Feature-Request#4]
![image](https://user-images.githubusercontent.com/33010254/89721975-4e4ac200-d998-11ea-9bd5-b159ef014249.png)

i would like to see CHallenge creating a new feature where I can deliver a challenge by sending 1 or X amount of CHL (OR EOS!) to challengedac or challengeyou with any eos account, and in the memo we could compress all necessary data about the challenge with many parameters like min and max reward all being defaulted to a one size fits all setting, so we ONLY have to put in the Radius in KM or Diameter of the challenge, from the ZIP CODE, and the ZIP CODE would be the next part, because no one want to give up their addresses! But a zip code or a BLOCK would be easier.  

TLDR: Allow us to create challenges by sending at least 1 CHL or 1 EOS to challengeyou or challengedac with only 3 pieces of information: zipcode - radius of challengezone - and finally the image link , youtube link and any text instructions that would be needed, as a TASK would be assumed if there was test, a GEO challenge assumed if there was a zip code or gps coordinates!
